prow/jobs/kyma-project/runtime-watcher/runtime-watcher.yaml

```diff
@@ -96,3 +96,61 @@ postsubmits:
           - name: signify-secret
             secret:
               secretName: signify-dev-secret
+    - name: release-runtime-watcher-build
+      annotations:
+        description: "build runtime watcher release image by semantic version tag"
+        owner: "jellyfish"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "watcher-build-release"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-sa-kyma-push-images: "true"
+        preset-signify-prod-secret: "true"
+      always_run: true
+      skip_report: false
+      decorate: true
+      cluster: trusted-workload
+      max_concurrency: 10
+      branches:
+        - ^\d+\.\d+\.\d+(?:-.*)?$
+      spec:
+        containers:
+          - image: "eu.gcr.io/sap-kyma-neighbors-dev/image-builder:v20230313-8dfce5f0b-buildkit"
```

Comment:
For what you need a buildkit builder? Can you use kaniko builder instead?

Reply:
@dekiel What is the difference between the two?

Reply:
Buildkit builder was added to make possible building of images for different architectures like arm64. Later the buildkit image was enhanced with bash and used for some very specific build process. Both of this use cases are deprecated and won't be supported in ADO backend. I'm asking to know if this is some special use case that you need a buildkit version.